diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter3/BlockStructureBracesWarnTest.kt

```diff
@@ -0,0 +1,119 @@
+package org.cqfn.diktat.ruleset.chapter3
+
+import com.pinterest.ktlint.core.LintError
+import org.cqfn.diktat.ruleset.constants.Warnings
+import org.cqfn.diktat.ruleset.rules.BlockStructureBraces
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.ruleset.utils.lintMethod
+import org.junit.Test
+
+class BlockStructureBracesWarnTest {
+
+    private val ruleId = "$DIKTAT_RULE_SET_ID:block-structure"
+
+    @Test
+    fun `check if with else if`() {
+        lintMethod(BlockStructureBraces(),
+                """
+                    |fun foo() {
+                    |    if (x > 0 && x != 10) {
+                    |        bar() 
+                    |        } else {
```

Comment:
```suggestion
                    |    } else {
```
Please don't discard indentation even inside this toy examples. Wrong indentation makes them less readable.